Tweak CFP docs and templates (#1660)
writethedocs/www: docs/conf/portland/2022/cfp_email_templates.rst

{% if cfp_variables['print_templates'] and not flagpostconf %}

.. contents::
   :local:
   :depth: 1
   :backlinks: none

CFP  - Acceptance template
~~~~~~~~~~~~~~~~~~~~~~~~~~

Tickets:
   {{cfp_variables['ticket']}}

--------------------------------------

Subject:
   WTD {{city}} {{year}} -- talk accepted!

::

   Hi {name},

>    The Write the Docs talk selection committee has just wrapped up the review process and we'd love it if you could join us as a speaker!

Reviewer: ```suggestion
   The Write the Docs talk selection committee has finished the review process and we'd love it if you could join us as a speaker!
```
(simpler)

   Write the Docs {city} is held in **{city} on {date}**.

>    We think your '{title}' talk would be a great fit for the conference. We'd love to have you prepare it for a **30-minute** time slot.

Reviewer: ```suggestion
   We think your talk: '{title}' would be a great fit for the conference. We'd love to have you prepare it for a **30-minute** time slot.
```

variables can get funky, I think this structure would make more sense

>    So, from here, your first step is to reply to this email as soon as possible, to confirm both of the following:

Reviewer: ```suggestion
  Please **reply to this email as soon as possible**, to confirm both of the following:
```

Front-loading the sentence with what needs to happen!

>    * You're still interested in presenting in {city} on {date}. We'll confirm the specific day and time for your talk at a later date.

Reviewer: ```suggestion
   * You're still interested in presenting for {city} on {date}. We'll confirm the specific day and time for your talk at a later date.
```

we might need a different template for virtual confs — but changing this preposition is a smol workaround

   * You'll be attending most of the conference. So much of the value of Write the Docs comes from the community interactions, so it's really important to us that our speakers are around to participate.

   Finally, get your free speaker ticket at: {{cfp_variables['ticket']}} (If you already purchased a ticket, let us know, and we will issue you a refund)

   Okay, with all that out of the way, it's time for the fun part – preparing your talk! To make sure everybody's on the same page, here are a few important things to keep in mind:

>    * Remember that one of the biggest strengths of the Write the Docs community is that we come from a huge variety of professional and personal backgrounds. When you're writing your talk (just like you're when writing documentation), think about the diverse needs and interests of your audience, avoid (or define) any jargony language, and make sure you clearly express what people are going to learn from your talk.

Reviewer: ```suggestion
   * Remember that one of the biggest strengths of the Write the Docs community is that we come from a huge variety of professional and personal backgrounds. When you're writing your talk (just like when you're writing documentation), think about the diverse needs and interests of your audience, avoid (or define) any jargon-y language, and make sure you clearly express what people are going to learn from your talk.
```
typos
